EMIP/1-1000/EMIP00130.xml

```diff
@@ -64,7 +64,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <note>(Budge, IV; Princeton Ethiopic 41, ff. 55v–56r; MYS 1, ff. 7rv).</note>
                     </msItem>
                <msItem xml:id="ms_i2"><locus from="10r" to="24v"/>
-                        <title>Prayer to Mary, “I take refuge.”</title>
+                 <title ref="LIT6161BakwelluH">Prayer to Mary, “I take refuge.”</title>
```

Comment:
If the record for work is filled properly, you do not need to include in addition "Prayer to Mary" etc.